internal/pkg/externalplugins/blunderbuss/blunderbuss.go

```diff
@@ -102,6 +103,15 @@ func HandlePullRequestEvent(gc githubClient, pe *github.PullRequestEvent,
 	repo := &pe.Repo
 	opts := cfg.BlunderbussFor(repo.Owner.Login, repo.Name)
 
+	// Wait a few seconds to allow other automation plugin to apply labels.
+	gracePeriod := 5 * time.Second
+
+	if opts.GracePeriodDuration <= 0 {
```

Comment:
I think better move this validate into `config.go`

Reply:
ok

Reply:
and I wrote it wrong